18.01.2021 12:39:32 Checking for system tray...
18.01.2021 12:39:32 Starting pgAdmin4 server...
18.01.2021 12:39:32 Creating server object, port:62238, key:e384ccca-b534-4aec-802d-1463b0cccc0b, logfile:C:/Users/toluwani/.pgAdmin4.427924103.log
18.01.2021 12:39:32 Python Path: C:/Program Files (x86)/pgAdmin 4/v4/venv/Lib/site-packages;C:/Program Files (x86)/pgAdmin 4/v4/venv/DLLs;C:/Program Files (x86)/pgAdmin 4/v4/venv/Lib
18.01.2021 12:39:32 Python Home: C:/Program Files (x86)/pgAdmin 4/v4/venv
18.01.2021 12:39:32 Initializing Python...
18.01.2021 12:39:33 Python initialized.
18.01.2021 12:39:33 Adding new additional path elements
18.01.2021 12:39:33 Redirecting stderr...
18.01.2021 12:39:33 stderr redirected successfully.
18.01.2021 12:39:33 Initializing server...
18.01.2021 12:39:33 Webapp Path: C:/Program Files (x86)/pgAdmin 4/v4/web/pgAdmin4.py
18.01.2021 12:39:33 Server initialized.
18.01.2021 12:39:33 Starting Server Thread...
18.01.2021 12:39:33 Open the application code and run it.
18.01.2021 12:39:33 Set the port number, key and force SERVER_MODE off
18.01.2021 12:39:33 PyRun_SimpleFile launching application server...
18.01.2021 12:39:34 Application Server URL: http://127.0.0.1:62238/?key=e384ccca-b534-4aec-802d-1463b0cccc0b
18.01.2021 12:39:34 The server should be up, we'll attempt to connect and get a response. Ping the server
18.01.2021 12:40:11 Attempt to connect one more time in case of a long network timeout while looping
18.01.2021 12:40:17 Everything works fine, successfully started pgAdmin4.
